ci_build_stubs.sh

```diff
@@ -55,7 +55,7 @@ function generate_stubs {
 
     NO_DEBUG="true" \
     STUBS_VERSION="$TRAVIS_TAG" \
-    dotnet run $LEAN_DIR $RUNTIME_DIR $STUBS_DIR
+    dotnet run -v q $LEAN_DIR $RUNTIME_DIR $STUBS_DIR
```

Comment:
I've just ran this locally and I don't think it does what you hope it will do (it won't get rid of the ~300 "Generating ..." messages). I think the only way to silence those messages is to set the stubs generator's minimum logging level to `Error` if the `NO_DEBUG` environment variable is set to `true`. This can be done by changing `Level.Info` to `Level.Error` in https://github.com/QuantConnect/quantconnect-stubs-generator/blob/master/QuantConnectStubsGenerator/Program.cs#L29.